### PR TITLE
doc(setup tool): add description on scope of setup.init()

### PR DESCRIPTION
**Pull request by pabloalonso**
Update bonita-bpm-platform-setup.adoc to clarify scope of init
Related to [RUNTIME-822](https://bonitasoft.atlassian.net/browse/RUNTIME-822)

### CODE REVIEW

**Comment by educhastenier** [comment]
@pabloalonso why did you choose 2021.1 as the base branch, and not earlier?

**Comment by pabloalonso** [comment]
There is no special reason

**Comment by educhastenier** [comment]
> There is no special reason

Change should be contributed on the first *supported* version if it was already applicable

File: modules/ROOT/pages/bonita-bpm-platform-setup.adoc [hunk 1]
```diff
@@ -51,6 +51,7 @@ The script `setup` comes with 4 commands:
 It creates the database tables and pushes the initial configuration in the database. +
 This initial configuration is taken from the `platform_conf/initial` folder. +
 It is run by the global script `start_bonita`, but you can also insert it in your own scripts.
```
Comment (baptistemesta) [inline modules/ROOT/pages/bonita-bpm-platform-setup.adoc]:
```suggestion
It is run by the global script `start_bonita`, but you can also insert it in your own scripts. +
```

**Pull request merged**